support-frontend/assets/components/checkoutBenefits/checkoutBenefitsListData.tsx

```diff
@@ -15,8 +15,13 @@ const boldText = css`
 	font-weight: bold;
 `;
 
-type TierUnlocks = {
+const highlightText = css`
+	color: red;
```

Comment:
Better to use a colour from our [brand palette](https://guardian.github.io/csnx/?path=/story/source-foundations_palette--palette) than just the generic CSS red. Has there been any design input on this at all?